wallet/src/de/schildbach/wallet/ui/dashpay/ContactsBasedLiveData.kt

```diff
@@ -0,0 +1,35 @@
+package de.schildbach.wallet.ui.dashpay
+
+import androidx.lifecycle.LiveData
+import de.schildbach.wallet.WalletApplication
+
+abstract class ContactsBasedLiveData<T>(val walletApplication: WalletApplication,
```

Comment:
Good idea to refactor this code out to a new class, rather than have duplicate code in many classes.